content/v0.80.0.md

```diff
@@ -0,0 +1,478 @@
+* 3. [How to upgrade](#Howtoupgrade)
+* 4. [New features](#Newfeatures)
+	* 4.1. [ReelMagic support](#ReelMagicsupport)
+	* 4.2. [Mouse improvements](#Mouseimprovements)
+	* 4.3. [Ability to use local configs via Finder on macOS](#AbilitytouselocalconfigsviaFinderonmacOS)
+* 5. [General improvements](#Generalimprovements)
+	* 5.1. [`more.com` command](#more.comcommand)
+	* 5.2. [On-the-fly changes](#On-the-flychanges)
+	* 5.3. [Post-rendered screenshots](#Post-renderedscreenshots)
+	* 5.4. [More bundled OpenGL shaders](#MorebundledOpenGLshaders)
+* 6. [Bug and regression fixes](#Bugandregressionfixes)
+* 7. [Translation improvements](#Translationimprovements)
+* 8. [Documentation updates](#Documentationupdates)
+* 9. [Contributors](#Contributors)
+* 10. [Thank you](#Thankyou)
+
+<!-- vscode-markdown-toc-config
+	numbering=true
+	autoSave=true
+	/vscode-markdown-toc-config -->
+<!-- /vscode-markdown-toc -->
+
+
+##  2. <a name='Summary'></a>Summary
+
+ReelMagic DOS games are now playable thanks to the efforts of Jon Dennis, Chris
+Guthrie, and Joseph Whittaker, with testing and feedback from Vogons users Uka,
+DonutKing, and Vetz. Mouse emulation has been improved and now supports up to
+seven independent mice, enabling split-screen play in games like Settlers 1 and
+Settlers II. More settings, including video, audio, and MIXER effects, can now
+be changed on the fly. A new `--working-dir` command line flag allows DOSBox
+Staging to launch from a specific working directory. Repo packagers should note
+that `SDL2-image` is now a dependency.
+ 
+Read on for all the details!
+
+##  3. <a name='Howtoupgrade'></a>How to upgrade
+
+Get started by downloading the latest version and updating your configuration
+file:
+
+**Downloads:**
+
+* [Linux](/downloads/linux/)
+* [Windows](/downloads/windows/)
+* [macOS](/downloads/macos/)
+
+**Upgrade your configuration file:**
+
+1. Launch the new release.
+2. At the`Z:\>` prompt, run: `config -wc` to write the new configuration file. 
+
+Notes:
+
+- Existing settings will be retained, but extraneous comments will be dropped.
+
+- Starting with version 0.80, the configuration file is written in the lauguage
+  in which DOSBox Staging was started.
+- To write an English configuration file, launch with the `-lang en`
+  command-line flag (to start DOSBox Staging in English) and then (re-)write
+  your configuration file.
+
+
+##  4. <a name='Newfeatures'></a>New features
+
+
+###  4.1. <a name='ReelMagicsupport'></a>ReelMagic support
+
+DOS ReelMagic games are now playable thanks to Jon Dennis's
+(patch)[https://github.com/jrdennisoss/dosboxrm] with help from Chris Guthrie
+and Joseph Whittaker.
+
+See the `[reelmagic]` section in your configuration file to enable this. We
+suggest only enabling ReelMagic emulation on an as-needed basis.
+
+<video controls width=640>
+  <source src="/reelmagic_rtz.mp4" type="video/mp4">
+  Your browser does not support the <code>video</code> element.
+</video>
```

Comment:
Please add the video in `webm` format as well.

Reply:
Here's the current state of the Bash function to encode at mp4:

```shell
cap2mp4() {
  # Needs echo, cut, grep, GNU awk, ffmpeg, and ffprobe
  # Uses latest ffmpeg built from sources:
  # git clone --depth 1 https://github.com/markus-perl/ffmpeg-build-script.git && cd ffmpeg-build-script &&  ./build-ffmpeg --build

  if [[ "$#" -eq 0 ]]; then
    echo "Usage: $0 INPUT.AVI [FPS] [WxH] [SECONDS]"
    return 1
  fi
  input="$1"
  output="${input%.*}.mp4"

  # Set or get frame rate
  if [[ "$#" -ge 2 && "$2" -gt 15 ]]; then
    fps="$2"
  else
    AWK=$(command -v gawk || command -v awk)
    fps=$(ffprobe -show_streams "$input" 2>&1 \
          | grep fps \
          | "$AWK" '{split($0,a,"fps")}END{print a[1]}' \
          | "$AWK" '{print $NF}')
  fi

  # Set or get the scaling dimensions
  if [[ "$#" -ge 3 ]]; then
    scale_dims="-s $3"
  fi

  # Set or get the duration
  if [[ "$#" -ge 4 ]]; then
    duration="-t $4"
  fi

  local crf="-qmin 7 -crf 23 -qmax 27"
  ffmpeg ${duration:-} -i "$input" -filter:v "fps=$fps" ${scale_dims:-} -sws_flags neighbor -sws_dither none \
         -c:v libx264 -pix_fmt yuv420p -vprofile high422 -vlevel 4.2 $crf -preset veryslow -g 600 -bf 16 -b_strategy 2 -b:v 128k -tune ssim \
         -c:a libfdk_aac -profile:a aac_he_v2 -vbr 3 -strict experimental \
         -movflags +faststart -y "$output"

  echo ""
  ls -1l "$output"
}
```

Reply:
Could we put this somewhere in the Wiki, maybe in some "documentation tips" section?